src/main/java/com/bynder/sdk/util/Utils.java

```diff
@@ -95,21 +96,32 @@ private static OkHttpOAuthConsumer createHttpOAuthConsumer(final String consumer
      * Creates an instance of {@link OkHttpClient}.
      *
      * @param consumer {@link OkHttpOAuthConsumer} instance.
+     * @param httpConnectionSettings settings for the http connection to Bynder
      *
      * @return {@link OkHttpClient} instance used for API requests.
      */
-    private static OkHttpClient createHttpClient(final OkHttpOAuthConsumer consumer) {
+    private static OkHttpClient createHttpClient(final OkHttpOAuthConsumer consumer, HttpConnectionSettings
+            httpConnectionSettings) {
```

Comment:
create a new method with the new argument

Reply:
same above, old method is unused now